xml/System.IO/Path.xml

```diff
@@ -1946,9 +1946,9 @@ Paths are resolved by calling the <xref:System.IO.Path.GetFullPath%2A> method be
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This method creates a temporary file with a .TMP file extension. The temporary file is created within the user's temporary folder, which is the path returned by the <xref:System.IO.Path.GetTempPath%2A> method.  
+ This method creates a temporary file with a .tmp file extension and returns the path to it. The temporary file is created within the user's temporary folder, which is the path returned by the <xref:System.IO.Path.GetTempPath%2A> method.  
   
- The <xref:System.IO.Path.GetTempFileName%2A> method will raise an <xref:System.IO.IOException> if it is used to create more than 65535 files without deleting previous temporary files.  
+ Previous to .NET 8.0, when using this method on Windows the <xref:System.IO.Path.GetTempFileName%2A> method will raise an <xref:System.IO.IOException> if it is used to create more than 65535 files without deleting previous temporary files. This limitation does not exist on other operating systems, nor does it exist on any operating system in .NET 8.0 and later versions.
```

Comment:
do we prefer to use something like ".NET 7.0 and earlier" (with the complications of .NET Framework) rather than caveating this way? i

Reply:
Yes, I like that better especially since .NET 8 hasn't released yet.